addendum.dict

```diff
@@ -407,6 +427,7 @@ Worple	𐑢𐑹𐑐𐑩𐑤
 Wynberg	𐑢𐑲𐑯𐑚𐑻𐑜
 Yeardsley	𐑘𐑸𐑛𐑟𐑤𐑦
 Youtube	𐑿𐑑𐑿𐑚
+Xbox 𐑧𐑒𐑕𐑚𐑭𐑒𐑕
```

Comment:
I'd prefer 𐑧𐑒𐑕𐑚𐑪𐑒𐑕 to conform to principle 14 at https://www.shavian.info/spelling/.